src/memefactory/styles/base/colors.clj

```diff
@@ -45,7 +45,7 @@
                     :busy-grey "#5c5c5c"
                     :rank-yellow "#ffd800"
                     :yellow "#ffea00"
-                    :cancel-button "#d0e5e8"
+                    :cancel-button "#ff0094"
```

Comment:
Why not using `:pink` we already have here?

Reply:
yeah, i'll just use the pink